rados/ioctx.go

```diff
@@ -114,12 +114,12 @@ func (ioctx *IOContext) Pointer() unsafe.Pointer {
 //  void rados_ioctx_set_namespace(rados_ioctx_t io,
 //                                 const char *nspace);
 func (ioctx *IOContext) SetNamespace(namespace string) {
-	var c_ns *C.char
+	var cns *C.char
```

Comment:
I find it easier to read and identify as a C allocated object, if the character after the `c` is upper case, like `cNS` and later `cOid` or `cOID`. But I don't care too much, no need to push another version because of that.

Reply:
Thanks. I will try to keep that in mind for next time. I'm pretty sure there'll be a next time. :-p

Reply:
You said exactly the same last time... 😉
https://github.com/ceph/go-ceph/pull/520#discussion_r659856907

Reply:
At least I'm consistent. ;-)